### DISCUSSION TITLE
Nose Wheel Taxi Light

### DISCUSSION THREAD

**Discussion by GB014934** [flybywiresim/aircraft#4885] [General]
I have the latest development version - updated 09:00 (UK Time) 24/5/2021.

When the taxi light is on should it turn as the nose wheel is steered ?  It does not seem to turn on my system - Am I missing something  ??

Trevor

**Comment by ExampleWasTaken** [comment]
If you report things not to be realistic, be sure to attach documentation or any other kind of references to the real aircraft.

The nose wheel light does not turn with the nose wheel on the real aircraft.

**Reply by GB014934** [reply]
Many thanks for the reply. I think I got confused with the 737 on Prepar3d.  The Aerosoft A320 taxi light on Prepar3d behaves just as the FBW version. Perhaps I should have checked !!
